experimental/packages/opentelemetry-sdk-metrics/test/export/utils.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AggregationSelector,
+  AggregationTemporalitySelector,
+  InstrumentType,
+  MetricReader
+} from '../../src';
+import * as assert from 'assert';
+
+
+/**
+ * Check if AggregationSelectors behave in the same way.
+ * @param reader
+ * @param expectedAggregationSelector
+ */
+export function assertAggregationSelector(reader: MetricReader, expectedAggregationSelector: AggregationSelector) {
+  assert.strictEqual(reader.selectAggregation(InstrumentType.COUNTER), expectedAggregationSelector(InstrumentType.COUNTER));
```

Comment:
Would it be worth reducing the duplicity here?

```suggestion
  const instrumentTypes = [ InstrumentType.COUNTER, ...];
  for (const instrumentType of instrumentTypes) {
    assert.strictEqual(reader.selectAggregation(instrumentType), expectedAggregationSelector(instrumentType));
  }
```

Same can be applied to `assertAggregationTemporalitySelector`.

Reply:
I initially kept it so that it is possible to see the exact line when the test fails. But I can also change it if you'd like. :slightly_smiling_face:

Reply:
Well, we can add assertion messages if we would like to know which aggregation of the instrument type is mismatching. In this way, we can share most of the lines between `assertAggregationSelector` and `assertAggregationTemporalitySelector`.

I'd find this can be easier to read if we have many items of this kind of enums -- since their steps are identical.

Reply:
Something like this maybe

```suggestion
  const instrumentTypes = [ InstrumentType.COUNTER, ...];
  for (const instrumentType of instrumentTypes) {
    assert.strictEqual(
      reader.selectAggregation(instrumentType),
      expectedAggregationSelector(instrumentType),
      `incorrect reader aggregation selection for ${InstrumentType[instrumentType]}`
    );
  }
```

Reply:
Done in [f0581e7](https://github.com/open-telemetry/opentelemetry-js/pull/3225/commits/f0581e7dfbfeaf880beb6fd950e47d5b0b134c26). :slightly_smiling_face: Thanks for the feedback, I'll keep it in mind for future tests. :slightly_smiling_face: